### ISSUE TITLE
Having a (any?) controller plugged into your PC trips the auto-kick

### ISSUE THREAD

**Issue by Jolly-66** [tgstation/tgstation#71980]
## Round ID:
Tested locally.

## Reproduction:
Be an unsuspecting fool.
Plug your controller into your PC.
Either using it or charging it.
Leave it plugged in.
Go play SS13:

![image](https://user-images.githubusercontent.com/70232195/207465645-b2afbc4e-f395-499c-a1de-7ef9de1bd9ab.png)

For reference, I have a Nintendo Switch controller. Don't know if this happens with other controllers.

EDIT: Should add this is a somewhat uncommon occurrence that happens to people. This isn't something we'd suspect to cause us to error out of the game, as player-facing it appears to fail to log us in.
For reference, see this  message posted in #help https://discord.com/channels/326822144233439242/543879244594413618/1051240928213479575

**Comment by ShizCalev** [comment]
Related - #70801

I'd assume the reason you'd be kicked for an idle switch controller specifically is due to it sending Gyro inputs, and byond thinking they're actual commands.

**Comment by Potato-Masher** [comment]
A friend had issues with this recently. We suspect any damaged controller might cause this as well, with stickdrift causing constant inputs and whatnot.

Would there be a way to let the user know that an active controller might be causing them to be kicked before they're fully booted out? It would save players a lot of time on troubleshooting if this was possible.

**Comment by Jolly-66** [comment]
IMO having something incredibly visible that just says "You've been kicked from the game" would partially remedy this, but it'll spill into admin territory with "why was I kicked?". 

You also need to make it visible enough to be beyond the "you're experiencing lag" banner that appears in chat.

**Comment by Siefer21** [comment]
I have a 360 pro controller plugged in 24/7 and never have ran into this sort of issue

**Comment by lessthnthree** [comment]
I wouldn't say uncommon. We get at least one ticket a month or more people getting kicked from having a controller plugged in. We could change the error message to be more clear about it.

**Comment by ShizCalev** [comment]
> I have a 360 pro controller plugged in 24/7 and never have ran into this sort of issue

If you actually use the joysticks while ingame, it will trigger. 
It's triggering w/ an idle switch controller since the controller has gyro inputs, so to the game, you're ALWAYS sending inputs with it.

**Comment by ShizCalev** [comment]
This is moooooore or less a duplicate report of #https://github.com/tgstation/tgstation/issues/70801 , gonna go ahead and close.
